src/core/CMakeLists.txt

```diff
@@ -94,6 +53,36 @@ install(
 )
 
 if (WITH_CUDA)
+  find_library(CUDA_NVRTC_LIBRARIES nvrtc
+    PATHS ${CUDA_TOOLKIT_ROOT_DIR}
+    PATH_SUFFIXES lib lib64 targets/x86_64-linux/lib targets/x86_64-linux/lib/stubs)
+
+  add_library(
+    tc_core_cuda
+
+    SHARED
+
+    cuda/cuda.cc
```

Comment:
how about using a regexp ? :) Then we will make sure things work in a similar way with both buck and cmake